API/Controllers/ReaderController.cs

```diff
@@ -467,6 +467,7 @@ public async Task<ActionResult<ProgressDto>> GetProgress(int chapterId)
             progressBookmark.SeriesId = progress.SeriesId;
             progressBookmark.VolumeId = progress.VolumeId;
             progressBookmark.PageNum = progress.PagesRead;
+            progressBookmark.LastModifiedUtc = progress.LastModifiedUtc;
```

Comment:
Remove this. IEntityDate does this for you already.

Reply:
I checked, it doesn't, probably because progressBookmark is instantiated directly instead of being read from the database.